### PR TITLE
Fix compute fre order and bump ocean lib

### PR DESCRIPTION
**Pull request by bogdanfazakas**
Fixes # .

Changes proposed in this PR:

- bump ocean lib to 1.0.13
- fix fre approve before calling buyFromFreAndOrder

### CODE REVIEW

File: src/@utils/order.ts [hunk 1]
```diff
@@ -83,6 +88,8 @@ export async function order(
         exchangeContract: config.fixedRateExchangeAddress,
         exchangeId: asset.accessDetails.addressOrId,
         maxBaseTokenAmount: orderPriceAndFees.price,
+        baseTokenAddress: asset?.accessDetails?.baseToken?.address,
+        baseTokenDecimals: asset?.accessDetails?.baseToken?.decimals || 18,
```
Comment (EnzoVezzaro) [inline src/@utils/order.ts:92]: I think we should move '18' to its own variable. Not sure how many 18's we have in the code, but I've seen it a couple of times, we should consider move it to a variable.

**Review by EnzoVezzaro — APPROVED** [review]
LGTM! just what suggested in comment

**Review by kremalicious — APPROVED** [review]
Confirmed on fixed price compute dataset it can be started with this PR

File: src/@utils/order.ts (hunk 1 above)
Comment (kremalicious) [reply src/@utils/order.ts:92]: it's not about being optional, it's about the app not breaking when it is already rendering but that specific data is not there yet. For hardcoded `18` we should have some const somewhere, but also through #1516 might be not needed anymore

**Pull request merged**